content/docs/Other guides/monitoring-with-prometheus-loki-and-grafana.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Monitoring with Prometheus, Loki, and Grafana"
+linkTitle: "Monitoring with Prometheus, Loki, and Grafana"
+weight: 5
+date: 2021-08-01
+description: >
+  Configuring OpenCue with external monitoring services
+---
+
+This page describes how to stand up a sample OpenCue deployment configured to send data to external
+monitoring services.
+
+The sandbox deployment described here utilizes:
+
+*   Prometheus for collecting metrics from the OpenCue database and scheduler.
+*   Loki for collecting logs from all OpenCue components.
+*   Grafana to provide an interface for querying and displaying the data send by the others.
+
+## Before you begin
+
+*  Follow our quick start guide, which will walk you through standing up a basic OpenCue
```

Comment:
Maybe add a link to /docs/quick-starts/ here for people who aren't familiar with the quick starts.

Reply:
Meant to do that, done.